docs/src/whatsnew/latest.rst

```diff
@@ -150,7 +154,9 @@ This document explains the changes made to Iris for this release
 🚀 Performance Enhancements
 ===========================
 
-#. N/A
+#. `@wjbenfold`_ resolved an issue that previously caused regridding with lazy
+   data to take significantly longer than with real data. Relevant benchmark
+   shows a time decrease from >10s to 625ms. (:issue:`4280`, :pull:`4400`)
```

Comment:
@wjbenfold I assume that you're cheekily slipping this forgotten `whatsnew` entry into the mix?

Reply:
Can you cite the relevant benchmark somehow?

Reply:
If I recall correctly, I was moving it from "bug fixes" to "performance enhancements". Looks like I just duplicated it though (or at least once merges were all resolved that's where we got to). Where do you reckon it belongs?

Reply:
@wjbenfold I reckon `Performance Enhancements` is the right section, but drop it from `Bugs Fixed`... we tend not to duplicate entries and I'd see it more as an optimistation rather than a bug fix IMHO